test/endpoint_test.go

```diff
@@ -62,7 +63,7 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func createTx(privateKey string, address string, utxo NodeUnspentUtxo) (*bt.Tx, error) {
+func createTx(privateKey string, address string, utxo NodeUnspentUtxo, fee ...uint64) (*bt.Tx, error) {
```

Comment:
For one tx you can have only one fee.
I suggest to change this to `createTx(privateKey string, address string, utxo NodeUnspentUtxo, fee uint64)`